tests/ethpm/test_package.py

```diff
@@ -67,4 +67,3 @@ def test_package_object_has_version_property(valid_lockfile):
 def test_package_has_custom_str_repr(valid_lockfile):
     current_package = Package(valid_lockfile)
     assert current_package.__repr__() == "<Package wallet==1.0.0>"
-    
```

Comment:
@njgheorghita It's a bit hard to tell from the diff what this change was.  If the newline at the end of the file was removed, in the future it would be to avoid that kind of a whitespace change.  Practically speaking, it doesn't really matter if a newline is at the end of a file but a lot of editors add it anyway to be POSIX compliant.  It's best just to leave them in so there aren't a lot of commits that undo and redo the same edits.

Reply:
@davesque Cool, good to know. If I remember correctly, flake8 was yelling at me b/c of the newline at end of file. But I was confused as to why the newline was there in the first place (if flake8 yelled now, it should have yelled earlier??)

Reply:
@njgheorghita Hmm...I would wager that flake8 was complaining about multiple newlines at the end of the file.  Here's a more explicit example of what I'm talking about:
```
$ printf 'foo = "bar"' > test.py
$ flake8 test.py
test.py:1:12: W292 no newline at end of file

$ printf 'foo = "bar"\n' > test.py
$ flake8 test.py    # Generates no errors because file ends with newline

$ printf 'foo = "bar"\n\n' > test.py
$ flake8 test.py
test.py:2:1: W391 blank line at end of file
```

Reply:
Ahh ok got it, thanks!